core/src/main/scala/org/ensime/filewatcher/FileWatchService.scala

```diff
@@ -180,12 +182,8 @@ class FileWatchService { self =>
       log.trace(s"register ${dir} with WatchService")
     if (!observers.isEmpty) {
       val key: WatchKey = try {
-        dir.toPath.register(
-          watchService,
-          ENTRY_CREATE,
-          ENTRY_MODIFY,
-          ENTRY_DELETE
-        )
+        val events: Array[WatchEvent.Kind[_]] = Array(ENTRY_CREATE, ENTRY_MODIFY, ENTRY_DELETE)
```

Comment:
you need to run `sbt fmt` ... this is beyond the line limit.

Reply:
sbt fmt'ed